config/v1/types_infrastructure.go

```diff
@@ -1002,6 +1002,17 @@ type PowerVSPlatformStatus struct {
 	// Note: Currently only single-zone OCP clusters are supported
 	Zone string `json:"zone"`
 
+	// resourceGroup is the resource group name for new IBMCloud resources created for a cluster.
+	// The resource group specified here will be used by cluster-image-registry-operator to set up a COS Instance in IBMCloud for the cluster registry.
+	// More about resource groups can be found here: https://cloud.ibm.com/docs/account?topic=account-rgs.
+	// When omitted, the image registry operator won't be able to configure storage,
+	// which results in the image registry cluster operator not being in an available state.
```

Comment:
So if the image registry operator is going to be unavailable when this isn't set, what happens to clusters that are upgrading? Have we got a plan to block their upgrades until they configure this value (I guess we will have to backport the API?)

Reply:
IPI for PowerVS isn't GA'ed yet, so there are hardly anyone using in the production yet.. so do we really need to worry about the upgrade scenario?

@clnperez @mjturek @Prashanth684 wdyt?

Reply:
Hi @JoelSpeed, Did you get a chance to look at @mkumatag's response? @clnperez has also agreed to this opinion on upgrade.

Reply:
If this is behind a TechPreviewNoUpgrade feature gate then I think you're right, you won't need to worry about upgrade. IIUC the plan here is that this will always be set for new installs going forward? We should make sure to use a CEL validation to make sure this field is never cleared. Does it also need to be immutable?

Reply:
yes, we don't expect customer to change this value post creation.

Reply:
Added CEL validation as requested. PTAL!